_rules/image-no-text-0va7u6.md

```diff
@@ -36,24 +36,23 @@ This rule applies to any [visible][] [embedded image][].
 
 For the rendered image resource of the [image sources][] of each test target, at least one of the following is true:
 
-- the image resource contains text and it is [essential][] that the text is rendered with that specific presentation; or
-- the image resource does not contain text expressing anything in a [human language][]; or
-- the image resource contains text and the text is not a significant part of the visual content in the image; or
+- the [visible pixels][visible] of the rendered image resource contain text and it is [essential][] that the text is rendered with that specific presentation; or
+- the [visible pixels][visible] of the rendered image resource do not contain text expressing anything in a [human language][]; or
+- the [visible pixels][visible] of the rendered image resource contain text and the text is not a significant part of the visual content in the image; or
```

Comment:
I'm wondering if we should restrict also to "significant part of the visible pixels".
If everything from the image is hidden except the bit that contain text, then that text was maybe not the main part of the full image but is the main part of what's left and this is probably bad.

I'm not fully sure of this because there might be cases where this is OK and we don't want false positives, but every case where this is OK I can think of tend to have that below image with text showing actually become decorative (and pass the next item in the list)… So I  just leave it here…
(in any case, this only causes the rule to fail too often so it is OK).

Reply:
I agree, and will update accordingly.